sklearn/ensemble/bagging.py

```diff
@@ -65,7 +65,7 @@ def _parallel_build_estimators(n_estimators, ensemble, X, y, sample_weight,
             print("building estimator %d of %d" % (i + 1, n_estimators))
 
         random_state = check_random_state(seeds[i])
-        seed = check_random_state(random_state.randint(MAX_INT))
+        seed = random_state.randint(MAX_INT)
```

Comment:
Can you summarize the result of the discussion? I though that passing `random_state` to joblib parallel has to be avoided to avoid serializing a random_state object (which is (was?) costly).